apps/omg_db/lib/omg_db/models/payment_exit_info.ex

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.DB.Models.PaymentExitInfo do
+  @moduledoc """
+  DB model wrapper that is responsible for Payment (V1) Exit Info.
+  """
+
+  alias OMG.DB
+
+  @server_name OMG.DB.RocksDB.Server
+
+  @five_seconds 5000
+  @ten_seconds 10_000
+  @one_minute 60_000
+
+  def exit_info(utxo_pos, server_name \\ @server_name) do
+    {:ok, data} = DB.get(:exit_info, [utxo_pos], @five_seconds, server_name)
+    {:ok, hd(data)}
+  end
+
+  def exit_infos(utxo_pos_list, server_name \\ @server_name)
+      when is_list(utxo_pos_list) do
+    DB.get(:exit_info, utxo_pos_list, @ten_seconds, server_name)
```

Comment:
I am not 100% sure on the data size yet. 10 sec might be premature large?

Reply:
Can you make it optional in `DB.get` with 5 seconds (default GenServer call's timeout)

Reply:
okay. To introduce multiple optional args, I have to use `opts \\ []` instead of `timeout \\ 5000, server \\ @server_name` to allow optional args combination in `timeout` and `server`. (unless there is better way of doing this?)